azure-functions/acbs-function/mappings/schemas/amount.schema.ts

```diff
@@ -0,0 +1,4 @@
+import { z } from 'zod';
```

Comment:
Please strictly stick to existing azure functions pattern as agree this has improved the code quality but please use the existing implementation to keep the coding paradigm uniform.